sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp

```diff
@@ -4343,7 +4343,7 @@ namespace Azure { namespace Storage { namespace Blobs {
                   path.size() == 2 && path[0] == XmlTagName::k_Properties
                   && path[1] == XmlTagName::k_ContentMD5)
               {
-                ret.HttpHeaders.ContentHash.Value = Base64Decode(node.Value);
+                ret.HttpHeaders.ContentHash.Value = Azure::Core::Base64Decode(node.Value);
```

Comment:
Too bad, we shouldn't edit auto-generated code manually.

Can you open an issue and assign it to me to update the auto-generator?

Reply:
How can I remove the method definition in Azure::Storage then in this PR, when the auto-generated code is still calling it?

Can you update the auto-generator and add a commit to this PR directly?

Reply:
I think you can just merge this PR with these changes as is now. I'll update auto-generator later.

Reply:
Sounds good. Filed an issue: https://github.com/Azure/azure-sdk-for-cpp/issues/1299